cpp/autosar/src/rules/A8-4-7/InParametersForNotCheapToCopyTypesNotPassedByReference.ql

```diff
@@ -28,7 +28,7 @@ import codingstandards.cpp.CommonTypes as CommonTypes
 from Parameter v
 where
   not isExcluded(v, ClassesPackage::inParametersForNotCheapToCopyTypesNotPassedByReferenceQuery()) and
-  not v.getType() instanceof TriviallySmallType and
+  not v.getType() instanceof TriviallyCopyableSmallType and
```

Comment:
the smallest improvement suggestion, we can change the alert to match the same format as the other query/style guide (put the single quotes around code elements)

Reply:
Done.